### ISSUE TITLE
Can I ask u some question?

### ISSUE THREAD

**Issue by domiee13** [Distance10086/bug_report#1]
Can I ask u some question? I have some question about CVE request, please reply me soon..

**Comment by Distance10086** [comment]
Certainly! I'd be happy to try to help you with your questions about CVE requests. Could you provide more information about your specific questions or concerns?

**Comment by domiee13** [comment]
Thank you so much for fast responding bro.

I have some question: 
- How did you contact with this vendor ? And how long did they reply you ?
- How did you request 3 CVEs ?

**Comment by Distance10086** [comment]
If you believe you have discovered a vulnerability in a software or system, you can apply for a CVE identifier through one of the authorized CVE Numbering Authorities (CNAs). Here is a general overview of the process:

Identify the relevant CNA: The first step in applying for a CVE identifier is to determine which CNA is responsible for issuing identifiers for the software or system in which you have discovered a vulnerability. There are multiple CNAs, each with responsibility for different domains. You can find a list of CNAs and the domains they cover at the following link: https://cve.mitre.org/cve/cna.html
https://www.cve.org/ResourcesSupport/ReportRequest

Submit a request: Once you have identified the relevant CNA, you will need to submit a request for a CVE identifier. The CNA may have specific requirements for the request, such as formatting guidelines or information that must be included. It is important to follow these requirements carefully to ensure that your request is processed smoothly.

Wait for a response: After you have submitted your request, the CNA will review it and determine whether a CVE identifier should be issued. This process can take some time, depending on the volume of requests and the resources available to the CNA.

Communicate the vulnerability: Once you have received a CVE identifier, it is important to communicate the vulnerability to the appropriate parties, such as the vendor or maintainer of the affected software or system. This will help to ensure that the vulnerability is promptly addressed and mitigated.

It is worth noting that not all vulnerabilities will be eligible for a CVE identifier. The CNA will consider factors such as the potential impact of the vulnerability and the likelihood that it will be exploited when deciding whether to issue an identifier.

**Comment by domiee13** [comment]
So, we will not contact with vendor, right ?

**Comment by Distance10086** [comment]
Correct. It is important to contact the vendor or project that maintains the affected product before applying for a CVE. This allows the vendor or project to assess the issue and determine if it is eligible for a CVE. It also allows the vendor or project to take any necessary steps to mitigate the vulnerability or exposure and protect their users.

If you discover a vulnerability or exposure and do not contact the vendor or project, it is possible that the issue will not be properly addressed and could result in harm to users or organizations. Therefore, it is important to follow responsible disclosure practices when reporting vulnerabilities or exposures, which generally involve contacting the vendor or project and giving them an opportunity to address the issue before it is made public.

**Comment by domiee13** [comment]
How did you contact with this source code's owner ? And how long did they reply ?

**Comment by Distance10086** [comment]
To contact the owner of a source code, you will need to determine who the owner is and how they prefer to be contacted. Here are some steps you can take to identify and contact the owner of a source code:

Look for contact information in the source code itself. Many source code repositories include information about the owner or maintainer of the code in a README file or other documentation.

Check the website or online presence of the project or organization associated with the source code. There may be information about how to contact the owner or maintainer of the code on the project website or social media pages.

If you are unable to find contact information in the source code or on the project website, you can try searching online for more information about the project or organization. This may include searching for the name of the project or organization on search engines or social media platforms.

If you are still unable to find contact information for the owner of the source code, you may need to try contacting the hosting service or platform where the source code is hosted. They may be able to provide you with information about how to contact the owner or maintainer of the code.

The length of time it takes to receive a response can vary depending on the owner or maintainer of the source code and their availability. It's a good idea to allow enough time for them to review your message and respond appropriately.

I hope this information is helpful. If you have any other questions about contacting the owner of a source code, please don't hesitate to ask.

**Comment by domiee13** [comment]
Thanks for your answer. But Im asking about specify people (oretnom23), because I can't contact with him : (
